pkg/cmd/server/api/types.go

```diff
@@ -418,7 +421,7 @@ type ServiceAccountConfig struct {
 
 	// MasterCA is the CA for verifying the TLS connection back to the master.  The service account controller will automatically
 	// inject the contents of this file into pods so they can verify connections to the master.
-	MasterCA string
+	MasterCA *string
```

Comment:
pretty sure we should have done this as a pointer originally, to distinguish between missing, intentionally set to nothing, and present